saleor/graphql/product/mutations/products.py

```diff
@@ -250,6 +256,10 @@ class Meta:
         error_type_class = CollectionError
         error_type_field = "collection_errors"
 
+    @classmethod
+    def post_save_action(cls, info, instance, cleaned_input):
```

Comment:
Do you need this empty method?

Reply:
Yes, `CollectionUpdate` inherits from `CollectionCreate`, so I needed to clear this method to avoid triggering `PRODUCT_UPDATED` webhook when collection is updated.

Reply:
So in that case I would call the product_updated inside a CollectionCreate.perform_mutation
or at least add the comment why we have empty method here <- which imho is not the best way.
@fowczarek  What do you think?

Reply:
Please correct me if im wrong, but `CollectionUpdate` also inherits `perform_mutation` from `CollectionCreate`.

Reply:
Oh, yes you're right. I missed thath!

Reply:
But at least comment why we have empty method here